build_scripts/build_usd.py

```diff
@@ -1135,6 +1297,24 @@ def InstallTIFF(context, force, buildArgs):
 def InstallPNG(context, force, buildArgs):
     with CurrentWorkingDirectory(DownloadURL(PNG_URL, context, force)):
         macArgs = []
+        if context.targetIos:
```

Comment:
can we just skip all this on ios instead of fixing libtiff for them? I can't think of any reason we'd want to create these utilities.